ui/src/milkdropvisualizer/VisualizerDialog.js

```diff
@@ -0,0 +1,110 @@
+import React, { lazy, Suspense } from 'react'
+import {
+  createMuiTheme,
+  makeStyles,
+  ThemeProvider,
+} from '@material-ui/core/styles'
+import Dialog from '@material-ui/core/Dialog'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import IconButton from '@material-ui/core/IconButton'
+import Typography from '@material-ui/core/Typography'
+import CloseIcon from '@material-ui/icons/Close'
+import Fade from '@material-ui/core/Fade'
+import { useSelector, useDispatch } from 'react-redux'
+import { showMilkdropVisualizer } from '../actions'
+import useCurrentTheme from '../themes/useCurrentTheme'
+const MilkDropVisualizer = lazy(() => import('./MilkDropVisualizer'))
+
+const useStyles = makeStyles((theme) => ({
+  dialog: {
+    zIndex: '5 !important',
```

Comment:
z-index of MUi dialog is 1300 so this is done so as you can still see the Jinkle player and, also marginBottom of 80px which is the height of the Jinkle player